src/test/java/org/broadinstitute/hellbender/cmdline/CommandLineProgramIntegrationTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * The MIT License (MIT)
```

Comment:
Remove the license from this file. GATK4 has a repository-wide BSD 3-clause license.

Reply:
Removed (sorry, my IDE attach sometimes old LICENSE config...)